modules/server_processes_manager/README.md

```diff
@@ -0,0 +1,65 @@
+# Server Processes Manager
+
+## Purpose
+
+The server processes manager module is used to get information about processes that were 
+launched asynchronously by other LORIS modules
+
+## Intended Users
+
+The only user that should have access to this module is the person in charge of
+the administration of the LORIS website
+
+## Scope
+
+This module displays the information relevant to the asynchronous processes
+(active and inactive) launched by other modules. The properties displayed for
+each process are as follows:
+
+- The internal LORIS process ID for the process
+- Pid: the Unix process ID of the command that was launched asynchronously.
+- Type: type of process (only mri_upload is currently implemented).
+- Stdout file: full Unix path of the Output log file on the LORIS server 
+(STDOUT output stream for the process).
+- Stderr file: full Unix path of the Error log file on the LORIS server 
+(STDERR output stream for the process).
+- Exit code file: full Unix path of the file on the LORIS server that will 
+contain the exit code of the process once it finishes.
+- Exit code: Unix exit code for the process. This column will be empty until 
+the process terminates.
+- Userid: the LORIS username of the user that launched the asynchronous process.
```

Comment:
In the future, maybe it would also be worth adding the UNIX user that the process ran as (but not as part of this PR if that's not the way it currently works..)

Reply:
Might be useful, yeah. In another PR as you suggested.